challenges/objects-arrays.js

```diff
@@ -59,12 +91,23 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+for (i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`); 
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+for (i = 0; i < graduates.length; i++) {
+  if (graduates[i].university.includes("Uni")) {
```

Comment:
An alternative way to use this one would have been to use the universities array you already created with fewer lines of code. Still though, good job!